components/utilities/ai.js

```diff
@@ -0,0 +1,184 @@
+/* AI logic to play a double */
+// Need to handle logic if there are 3/4 of a kind
+// This leads to different combinations of doubles.
```

Comment:
it's a little cleaner to keep your comment styles consistent. For single lines i recommend just the double forward slashes. for multiple lines, the large /* */ comment style is preferred.

ex:
```javascript
// This is a single comment
const foo = bar;
```

```javascript
/*
 * This is a multi-line comment
 * So we prefer to use this style
 */
const foo = bar;
```

Reply:
I'll keep this in mind, I removed the multi-lines, but going forward I gotchu.